sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -86,3 +86,533 @@
 --    behavior: can take a long time.
 
 UPDATE `background_services` SET `require_once`='/library/MedEx/MedEx_background.php' WHERE `name`='MedEx';
+
+#IfNotTable form_eye_base
+CREATE TABLE `form_eye_base` (
+  `id`         bigint(20) NOT NULL COMMENT 'Links to forms.form_id',
+  `date` datetime DEFAULT NULL,
+  `pid`        bigint(20)   DEFAULT NULL,
+  `user`       varchar(255) DEFAULT NULL,
+  `groupname`  varchar(255) DEFAULT NULL,
+  `authorized` tinyint(4)   DEFAULT NULL,
+  `activity`   tinyint(4)   DEFAULT NULL
+) ENGINE = InnoDB;
+ALTER TABLE `form_eye_base`
+  ADD PRIMARY KEY `form_link` (`id`);
+ALTER TABLE `form_eye_base`
+  CHANGE `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT;
```

Comment:
These ALTER TABLE statements should be removed and these things should be included in the CREATE TABLE call. Also should do the same for all the other tables.

Reply:
Not sure how to do this.  When I try I get a mysql error, but this way works.  Please adjust if there is a better way.  Next review has another unique index too.

Reply:
Can't you just do what you did in the old form_eye_mag table regarding the id column and the primary key?